anyway/infographics_dictionaries.py

```diff
@@ -1,7 +1,36 @@
+from flask_babel import _
+
 driver_type_hebrew_dict = dict(
     professional_driver="נהג מקצועי", private_vehicle_driver="נהג פרטי", other_driver="לא ידוע"
 )
 head_on_collisions_comparison_dict = dict(
     head_to_head_collision="התנגשות חזית בחזית", others="אחרות", head_to_head="חזיתיות"
 )
 english_accident_severity_dict = {1: "fatal", 2: "severe", 3: "light"}
+
+accident_type_dict = {1: _("Pedestrian injury"),
```

Comment:
Here the `accident_type_dict` contains the result of the call to _(), right?
I think the user of the dict want to issue the call to _() at the time it of access to the dict. Maybe add a static function that will get the value from the dict and issue the call to _()?